content/03.results.md

```diff
@@ -4,12 +4,28 @@
-Next, we created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) with continuous integration to ensure analysis reproducibility.
-We developed methods for generating consensus SNVs from Lancet, Mutect2, and Strelka2 as well as consensus copy number altered regions from MantaSV, CNVkit, and ControlFREEC (**STAR Methods**). 
-This resulted in high-confidence SNVs, gains, losses, amplifications, and deep deletion calls used as input for all downstream analyses.
-We additionally created a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@url:https://manubot.org/ ; @doi:10.1371/journal.pcbi.1007128] integration to enable manuscript creation using Markdown within GitHub.
-
-We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project.  
+We maintained a data release folder on Amazon S3 containing merged files for each analysis, downloadable from the GitHub repository or open access CAVATICA project (See **Data and code availability** section).
+As new analytical products that we expected to be used across multiple analyses were produced (e.g., tumor mutation burden calculations) or issues with the data were identified, new releases were made available in a versioned manner.
+
+A key innovative feature of this project was the open contribution model used for both analyses (i.e., analytical code) and scientific manuscript writing. 
+We created a public Github analysis repository (https://github.com/AlexsLemonade/OpenPBTA-analysis) to hold all code associated with analyses downstream of the Kids First Data Resource Center workflows and a GitHub manuscript repository (https://github.com/AlexsLemonade/OpenPBTA-manuscript) with ManuBot [@doi:10.1371/journal.pcbi.1007128] integration to enable real-time manuscript creation using Markdown within GitHub.
+
+Additions to the manuscript often started with proposing a specific analysis in the analysis repository (for example, immune cell deconvolution of bulk RNA-Seq data).
+First, a potential contributor proposed an analysis by filing an issue on the GitHub analysis repository. 
+Next, organizers for the project, or other contributors with expertise, had the opportunity to comment on the proposal (**Figure {@fig:Fig1}C**). 
+The contributor then forked the analysis repository (or make a copy for the purpose of adding code changes) and added their proposed analysis to their fork. 
+They then filed a pull request on GitHub, where they requested to add their proposed changes to the main repository. 
+All pull requests to the analysis repository underwent peer review by organizers and/or other contributors. 
+Analytical code reviews conducted by peer reviewers checked for correctness of analyses (**Figure {@fig:Fig1}C-D**) and maintainability and readability of code and documentation. 
```

Comment:
```suggestion
Analytical code reviews conducted by peer reviewers checked for analysis accuracy (**Figure {@fig:Fig1}C-D**) and maintainability as well as readability of code and documentation. 
```

Personal preference - when I write `(1 and 2) and (3 and 4)`, I like to use _not and_ between them. So: `(1 and 2) as well as `3 and 4)`.